frontend/src/components/bs-space/components/objects-table/components/action-cell/action-cell.tsx

```diff
@@ -0,0 +1,21 @@
+import { IconButton } from 'components/common/common';
+import { IconName } from 'common/enums/enums';
+
+const ActionCell = (
+  id: string,
+  name: string,
+  onObjectDownload: (objectId: string, fileName: string) => void,
+): JSX.Element => {
+  const handleDownload = (): void => {
+    onObjectDownload(id, name);
```

Comment:
```suggestion
    onObjectDownload(id);
```
find name in the store

Reply:
Doing this, I pass this name to insert into the store

Reply:
isn't fixed

Reply:
how should I find the name of the file in the store if I put it in there calling this function and not taking it from headers yet?

Reply:
if I don't pass it here - there is no name in the state

Reply:
you have an array of objects 🙂
you can find an object (ex. `find`) and take the name 👍